### PR TITLE
PR with Publisher Page and Model Markdown

### PR DESCRIPTION
**Pull request by DDDSSSTTT**
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms and Google's Privacy Policy at https://www.google.com/policies/privacy.

We check modified Markdown files for validity using [this](https://github.com/tensorflow/tfhub.dev/blob/master/.github/workflows/contributions-validator.yml) GitHub Workflow. You can execute the same checks locally by passing the file paths of modified Markdown files (relative to the `assets/docs` directory) to validator.py e.g. `python3 ./tools/validator.py google/google.md google/models/albert_base/1.md ...`.

### CODE REVIEW

File: assets/docs/steinding/steinding.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Publisher steinding
+Stein Ding
+
+[![Icon URL]](https://drive.google.com/file/d/160VWvcHH6reFYSt6OM9l_-JjQB8HvWUu/view?usp=sharing)
```
Comment (WGierke) [inline assets/docs/steinding/steinding.md:4]: It seems like this is not public. Could you double-check?

Comment (DDDSSSTTT) [reply assets/docs/steinding/steinding.md:4]: Hi WGierke, sorry for the mistake. Now Everyone can check this photo.

File: assets/docs/steinding/models/yolov5/1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Module steinding/yolov5/1
+
+Object detection model pretrained on YoloV5. 
+<!-- asset-path: https://drive.google.com/drive/folders/1UGiX2CpMVpgRZY1FsdfjOlV9LU9An2HG?usp=sharing -->
```
Comment (WGierke) [inline assets/docs/steinding/models/yolov5/1.md:4]: Could you host it as a tar.gz archive somewhere where the URL ends in `tar.gz`? Previous publishers e.g. used GCS

File: assets/docs/steinding/models/yolov5/1.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<!-- task: Object-detection -->
```
Comment (WGierke) [inline assets/docs/steinding/models/yolov5/1.md:5]: Please change the value to `image-object-detection`

File: assets/docs/steinding/models/yolov5/1.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<!-- network-architecture: regnety -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: Anti 996-->
```
Comment (WGierke) [inline assets/docs/steinding/models/yolov5/1.md:9]: Could you please choose a key from https://github.com/tensorflow/tfhub.dev/blob/master/tags/license.yaml? If none matches, please choose "custom" and copy the license to the bottom of this file.

**Review by WGierke — COMMENTED** [review]
Thank you for the contribution!
I left a few comments. Please let me know if you have any questions!

Also: If you submit `assets/docs/steinding/steinding.md` in a separate PR, the workflow tests will run automatically on every push of yours and will give you immediate feedback :)

File: assets/docs/steinding/models/yolov5/1.md (hunk 4 above)
Comment (DDDSSSTTT) [reply assets/docs/steinding/models/yolov5/1.md]: Done, changed to custom

File: assets/docs/steinding/models/yolov5/1.md (hunk 3 above)
Comment (DDDSSSTTT) [reply assets/docs/steinding/models/yolov5/1.md]: Done

File: assets/docs/steinding/models/yolov5/1.md (hunk 2 above)
Comment (DDDSSSTTT) [reply assets/docs/steinding/models/yolov5/1.md]: Done, now changed to some GCP storage url

File: assets/docs/steinding/models/yolov5/1.md (hunk 4 above)
Comment (WGierke) [reply assets/docs/steinding/models/yolov5/1.md]: Please also copy the license that applies to the bottom of the page

Comment (DDDSSSTTT) [reply assets/docs/steinding/models/yolov5/1.md]: License content is now added to the bottom

**Comment by DDDSSSTTT** [comment]
Hi WGierke, I just add the content of license to the markdown two days ago. Would you please have a look at that if you have the time?
Thanks!

**Comment by WGierke** [comment]
Hi DDDSSSTTT, please understand that we need to review licenses which deviate from the licenses from license.yaml. Unfortunately, I cannot give you an ETA on when the review of the license is completed.

**Comment by DDDSSSTTT** [comment]
> Hi DDDSSSTTT, please understand that we need to review licenses which deviate from the licenses from license.yaml. Unfortunately, I cannot give you an ETA on when the review of the license is completed.

OK, so if I am using a custom license here. How can I make it to pass the review. Should I do something more than copy/pasting the license at the bottom?

**Comment by WGierke** [comment]
If you want to use a custom license that is not one of the OSS licenses from licenses.yaml, we have to send the license to an internal review. There is nothing else you need to do since you already provided the license.

**Comment by WGierke** [comment]
[tfhub.dev](http://tfhub.dev/) has been converged with [Kaggle Models](https://kaggle.com/models). You can refer [here](https://www.kaggle.com/discussions/product-feedback/448425) for an update. Future contributions will be handled by the Kaggle team on https://kaggle.com/models. Thank you!

**Pull request closed without merging**